doc/conf.py

```diff
@@ -215,6 +215,12 @@
     "member-order": "bysource",
 }
 
+# sphinxcontrib.spelling options
```

Comment:
That presumably needs to be added to doc/requirements.txt.

Reply:
And to extensions? How does this even work?

Reply:
Its not added to our extensions or requirements, its not run as part of our docs build. I've run it by hand, here's the options if anyone else wants to run it by hand, and we may decide to enforce it in the near future.

Reply:
Oh ok, is there a reason we shouldn't add it as part of the doc build? It seems useful and you've built the extra dictionary?

Reply:
So I didn't want to add it as it still reports a bunch of warnings (we have a mix of American and British English, and still a lot of words that aren't in the extra dictionary), and I don't think its worth including it in the CI build until its Green. But it turns out that adding it to the `extensions` list doesn't actually run it as part of the normal build - it always need to be run manually/explicitly with `-b spelling`. So I'll add it to the requirements and extensions, but not as a separate CI step yet.

Reply:
So this actually [relies on](https://pyenchant.github.io/pyenchant/install.html#installing-the-enchant-c-library) `libenchant` which is not installed in the CI docker image. I don't think its worth rebuilding these until this is an actual CI requirement, so I've removed it from the extensions again - lets revisit if/when we want to enforce, and update the Docker images then.